README.md

```diff
@@ -32,6 +32,10 @@ wsl --install
 docker run --name RGO -e POSTGRES_PASSWORD=postgrespw -p 5432:5432 -d postgres
 ```
 
+```powershell
```

Comment:
Could there be a heading for this ? 
Can we add a line to ensure there are no duplicate containers using the same ports ?